Sources/TestingMacros/Support/Additions/FunctionDeclSyntaxAdditions.swift

```diff
@@ -142,11 +142,7 @@ extension FunctionParameterSyntax {
     // Enclose the base type declaration reference in a 1-element tuple, e.g.
     // `(<baseType>)`. It will be used in a member access expression below, and
     // some types (such as function types) require this.
-    let metatypeMemberAccessBase = TupleExprSyntax {
-      LabeledExprListSyntax {
-        LabeledExprSyntax(expression: baseTypeDeclReferenceExpr)
-      }
-    }
+    let metatypeMemberAccessBase = TupleExprSyntax(elements: [LabeledExprSyntax(expression: baseTypeDeclReferenceExpr)])
```

Comment:
I'd maybe add a comment mentioning why we're _not_ using a result builder here.

Reply:
I've actually broken this workaround out into a separate PR, now. See https://github.com/apple/swift-testing/pull/541. I only had it here temporarily as a quick way to validate it in CI.